src/MultiSelectOption.js

```diff
@@ -27,7 +27,10 @@ const MultiSelectOption = ({ label, active, disabled, onClick, className }) => (
             className={checkboxClassname}
             checked={active}
             label={label}
-            onChange={onClick}
+            onChange={(value, e) => {
```

Comment:
Wouldn't the un-used `value` trigger a ESLint violation?

Reply:
I think that under the hood the ESLint rule is configured like this:
`no-unused-vars: ['error', { args: 'after-used' }
So this means that you can have unused arguments **before** the ones you are using, but not **after** the last one you've used.
https://eslint.org/docs/rules/no-unused-vars#args

Reply:
Yeah as long as the last argument isn't unused it's fine.